js/ui/scroll_view/ui.scrollable.js

```diff
@@ -515,7 +511,7 @@ const Scrollable = DOMComponent.inherit({
                 : leftPosition;
 
             if(this._isRtlNativeStrategy()) {
-                scrollPosition.left += this._container().get(0).offsetWidth - this._container().get(0).clientWidth;
+                scrollPosition.left += this.container().offsetWidth - this.container.clientWidth;
```

Comment:
this.container.clientWidth -> this.container().clientWidth ?

Reply:
fixed in https://github.com/DevExpress/DevExtreme/pull/18560/commits/8704fb4a882b93bf28bbeb6506630887366c1c79